satpy/readers/viirs_edr_active_fires.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2019 SatPy Developers
+#
+# Author(s):
+#
+#   Rohan Daruwala <rdaruwala@wisc.edu>
```

Comment:
Please remove the author section here and instead add yourself to the AUTHORS file in the root of the repository. I'm trying to get away from individual authors in each module. They become out of date and not representative of the real authors.

Reply:
Done!